### PR TITLE
Removed raw format node after qemu/qsd 9.0.0

### PR DESCRIPTION
**Pull request by zhencliu**
ID: 2776

### CODE REVIEW

**Comment by zhencliu** [comment]
hi @aliang123 , would you please help to verify this patch? Thanks a lot.

**Comment by aliang123** [comment]
Ack for mirror cases:

- Blockdev_mirror_filternodename —-> qcow2->raw
- Blockdev_mirror_simple —---> raw -> raw
- Blockdev_mirror_to_rbd   → qcow2->rbd+raw

**Review by aliang123 — APPROVED** [review]
Acked-by: Aihua Liang <aliang@redhat.com>

File: provider/virt_storage/storage_volume.py [hunk 1]
```diff
@@ -180,16 +204,19 @@ def info(self):
         out["url"] = self.url
         out["path"] = self.path
         out["key"] = self.key
-        out["format"] = self.format.TYPE
+        # __hash__ uses it when adding a volume object
+        out["format"] = self._params.get("image_format", "qcow2") if self._params else None
```
[the remark this answers is not in the document]
Comment (zhencliu) [reply provider/virt_storage/storage_volume.py:208]: yes, when we add a volume object into a set in __init__, the set will do some checks if the volume has already been there, as you know there is no dup items in a set, so here __hash__ and __eq__ were defined for that purpose. At that moment, the _params is None.

**Comment by zhencliu** [comment]
@XueqiangWei would you please help to review this patch? Thanks.

**Review by XueqiangWei — APPROVED** [review]
LGTM,
Acked-by: xuwei@redhat.com

**Pull request merged**